projects/Mallard/src/components/article/html/article.ts

```diff
@@ -71,6 +79,9 @@ const renderArticleContent = (
                             </div>
                         `
                     }
+                    if (el.html.includes('•')) {
```

Comment:
do we need to handle `<ul>` tag?

Reply:
The styling for `<ul>` is already handled with `listStyles` in `lists.ts`.

Reply:
👍